core/server/proxy/src/main/java/alluxio/proxy/s3/S3RestServiceHandler.java

```diff
@@ -1217,9 +1217,15 @@ private Response getObject(final String bucket,
 
           Response.ResponseBuilder res = Response.ok(ris)
               .lastModified(new Date(status.getLastModificationTimeMs()))
-              .header(S3Constants.S3_CONTENT_LENGTH_HEADER, s3Range.getLength(status.getLength()))
-              .header(S3Constants.S3_CONTENT_RANGE_HEADER,
-                  String.format("%s/%s", range, status.getLength()));
+              .header(S3Constants.S3_CONTENT_LENGTH_HEADER, s3Range.getLength(status.getLength()));
+
+          // Check range
+          if (range != null) {
+            res.status(Response.Status.PARTIAL_CONTENT)
+                .header(S3Constants.S3_ACCEPT_RANGES_HEADER, S3Constants.S3_ACCEPT_RANGES_VALUE)
+                .header(S3Constants.S3_CONTENT_RANGE_HEADER,
+                    String.format("%s/%s", range.replace("=", " "), status.getLength()));
```

Comment:
will range be a random malformatted str without '=' ?

Reply:
Calling method `S3RangeSpec.Factory.create(range)` when building S3RangeSpec will check the format.

Reply:
yes but here it using the original string "range" to do the range.replace("=", " "),  if it's an  INVALID_S3_RANGE_SPEC from S3RangeSpec interpretation, then we should not return the Content-Range header to align with aws s3 behavior

Reply:
oh. I get it.